descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "invoice",
-      "version": "3.0",
+      "version": "3.1",
```

Comment:
why do we need to increase to 3.1?, 3.0 is already an increased version from 2.0

Reply:
There have been minor changes since it was increased from 2.0 to 3.0.

Reply:
yea I get that, changing the version numbers during different times is a little confusing. I think we need to update our release procedure with clear definitions